docs/core/tutorials/testing-with-cli.md

```diff
@@ -14,194 +14,158 @@ ms.assetid: 52ff1be3-d92e-4477-9c84-8c1771e87ab5
 
 # Organizing and testing projects with the .NET Core command line
 
-This tutorial follows [Getting started with .NET Core on Windows/Linux/macOS using the command line](./using-with-xplat-cli.md) to show how to go beyond simple "hello world" scenarios and pave the way for more advanced and well-organized applications.
+This tutorial follows [Getting started with .NET Core on Windows/Linux/macOS using the command line](using-with-xplat-cli.md) taking you beyond the creation of a simple console app to pave the way for the development of advanced and well-organized applications. After showing you how to use folders to organize your code, this tutorial shows you how to extend a console application with the [xUnit](https://xunit.github.io/) testing framework.
```

Comment:
comma before "taking"
instead of "to pave" etc., simply "to develop advanced and well-organized applications."